tests/Dfe.PlanTech.Infrastructure.Contentful.UnitTests/Content/Renderers/Models/PartRenderers/TableRendererTests.cs

```diff
@@ -0,0 +1,112 @@
+using System.Text;
+using Contentful.Core.Models;
+using Dfe.PlanTech.Domain.Content.Interfaces;
+using Dfe.PlanTech.Domain.Content.Models;
+using Dfe.PlanTech.Infrastructure.Contentful.Content.Renderers.Models;
+using Dfe.PlanTech.Infrastructure.Contentful.Content.Renderers.Models.PartRenderers;
+using Microsoft.Extensions.Logging.Abstractions;
+using TableCellRenderer =
```

Comment:
You shouldn't need all these `using CLASS = NAMESPACE.CLASS` references buddy, does it not work without them?